docker-compose.example.yml

```
@@ -0,0 +1,40 @@
+version: '3'
+
+services:
+  fhir:
+    depends_on:
+      - mongo
+      - redis
+    environment:
+      # Change this for public location of bulk-export-server this should be the FQDN and location of where the
+      # bulk-export container is made public to users. ex. https://abacus.example.com/bulk-export
+      PUBLIC_BULK_SERVER: http://localhost:3000
+      DB_HOST: mongo
+      DB_PORT: 27017
+      DB_NAME: bulk-export-server
+      REDIS_HOST: redis
+      REDIS_PORT: 6379
```

Suggest moving the environment variables for the default ports into the Dockerfile so they don't need to be here.

```suggestion
      DB_HOST: mongo
      REDIS_HOST: redis
```

---

Do I still need the `DB_NAME` here?

---

That's a good point. The default could be kept in the Dockerfile